contracts/mock_contract/src/lib.rs

```diff
@@ -0,0 +1,89 @@
+use cosmwasm_std::to_binary;
+use cosmwasm_std::Deps;
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::Binary;
+use cosmwasm_std::MessageInfo;
+use cosmwasm_std::StdError;
+use cosmwasm_std::Response;
+use cosmwasm_std::StdResult;
+use cosmwasm_std::DepsMut;
+use cosmwasm_std::{Env, entry_point};
```

Comment:
Bunch of clippy warnings.
`cargo clippy -- -D warnings`

Would you be opposed to having a pre-push hook installed?